src/realm/object-store/object_store.cpp

```diff
@@ -1054,6 +1057,12 @@ static void append_errors(std::string& message, std::vector<ObjectSchemaValidati
     }
 }
 
+static void append_line(std::string& message, const std::string& line)
```

Comment:
can we take `line` as a std::string_view rather than a `const std::string&` - it looks like you're always calling this with a const char* anyways, so that should avoid having to construct the temporary std::string for the `line` argument.